editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -2395,7 +2398,43 @@ Error EditorSceneImporterGLTF::_parse_animations(GLTFState &state) {
 
 				track->weight_tracks.resize(wc);
 
-				const int wlen = weights.size() / wc;
+				int wlen = weights.size() / wc;
+
+				const int expectedLen = times.size() * outputElements;
```

Comment:
We use `snake_case` for identifiers, so these should be `expected_len` and `output_elements`.

Reply:
Oops, sorry for that.